### ISSUE TITLE
could I use `global/static variable` in function?

### ISSUE THREAD

**Issue by fatfatson** [vnmakarov/mir#164]

**Comment by vnmakarov** [comment]
Yes, you can have bss or data inside the function (between func and endfunc).

Any label (including for bss and data) has module scope.  That is why it is possible to use data/bss in func.  The actual placement of data/bss relative to function address is undefined but they are not placed inside the function insns.

**Comment by fatfatson** [comment]
> Yes, you can have bss or data inside the function (between func and endfunc).
> 
> Any label (including for bss and data) has module scope. That is why it is possible to use data/bss in func. The actual placement of data/bss relative to function address is undefined but they are not placed inside the function insns.

any example for this? thanks.
maybe an existed variable in c/c++ or an mir global varaible
